### PR TITLE
Simplify config imports in cli package

### PR DESCRIPTION
**Pull request by ivov**
This PR sets a path alias for `config` to simplify imports in the `/cli` package.

Whitespace changes are autosave fixes from `.editorconfig`.

### CODE REVIEW

**Comment by janober** [comment]
Thanks a lot! Did you test this PR and did it work fine? Because for me it throws errors like:
```bash
(node:8) [MODULE_NOT_FOUND] Error Plugin: n8n: Cannot find module '@config'
```

Not sure anymore about the details but I think I also looked into that shortly around 1 or 2 years ago. Could be totally wrong, because of the long time, but I think it was maybe related to lerna?

**Comment by ivov** [comment]
Tested and working. Please try: `lerna clean -y && lerna bootstrap --hoist && npm run build`

export class Queue

**Comment by ivov** [comment]
TS can resolve aliased paths in source files and VSCode supports this out of the box as well, but TS does not emit JS with those aliased paths resolved. I added a module resolver with the [module-alias package](https://www.npmjs.com/package/module-alias), adjusting path resolution in our entry file at `packages/cli/bin/n8n`, which fixes this.

File: packages/cli/tsconfig.json [hunk 1]
```diff
@@ -19,7 +19,13 @@
 		"target": "es2017",
 		"sourceMap": true,
 		"emitDecoratorMetadata": true,
-		"experimentalDecorators": true
+		"experimentalDecorators": true,
+		"baseUrl": ".",
```
Comment (BHesseldieck) [inline packages/cli/tsconfig.json:23]: ist this supposed to be here?

**Review by BHesseldieck — APPROVED** [review]
just one remark 👍🏼

File: packages/cli/tsconfig.json (hunk 1 above)
Comment (ivov) [reply packages/cli/tsconfig.json:23]: As discussed, we're keeping it here, but there is an alternative without `baseUrl`.

**Comment by janober** [comment]
Great, just tested. Seems to work great. No idea if the command helped or the changes but all good.

Got merged.

**Pull request merged**